clients/egui/src/lib.rs

```diff
@@ -233,8 +235,11 @@ impl<'window> WgpuLockbook<'window> {
             self.context.request_repaint();
         }
 
-        out.egui = full_output.platform_output;
-        out
+        Output {
```

Comment:
assembling this at the end instead of mutating it along the way is a style change that helps me make sure I didn't forget to populate something